x-pack/plugins/observability_solution/observability/server/services/slo/find_slo.ts

```diff
@@ -42,14 +45,26 @@ export class FindSLO {
 }
 
 function mergeSloWithSummary(sloList: SLO[], sloSummaryList: SLOSummary[]): SLOWithSummary[] {
-  return sloSummaryList
-    .filter((sloSummary) => sloList.some((s) => s.id === sloSummary.id))
-    .map((sloSummary) => ({
-      ...sloList.find((s) => s.id === sloSummary.id)!,
-      instanceId: sloSummary.instanceId,
-      summary: sloSummary.summary,
-      groupings: sloSummary.groupings,
-    }));
+  return [
+    ...sloSummaryList
```

Comment:
I would appreciate if these two filters slo summaries could be captured in a semantic variable for readability

localSummaryList = sloSummaryList.filter...
remoteSummaryList = sloSummaryList.filter...

Reply:
Let's capture those in variables and then do the array merge with the variables.